hbase-client/src/main/java/org/apache/hadoop/hbase/security/SaslUtil.java

```diff
@@ -62,7 +65,8 @@ public boolean matches(String stringQop) {
 
   /** Splitting fully qualified Kerberos name into parts */
   public static String[] splitKerberosName(String fullName) {
-    return fullName.split("[/@]");
+    List<String> result = Splitter.onPattern("[/@]").splitToList(fullName);
```

Comment:
return Splitter.onPattern("[/@]").splitToStream(fullName).toArray(String[]::new);

Reply:
I saw your preference for using a stream to do toArray on the hbase-backup review, will apply the pattern to all such places, no problem